react/components/atoms/container-swipeable/container-swipeable.js

```diff
@@ -169,11 +174,15 @@ export class ContainerSwipeable extends PureComponent {
     _container = () => {
         return (
             <>
-                <Animated.View
-                    style={this._overlayStyle()}
-                    onStartShouldSetResponder={e => true}
-                    onResponderRelease={this.onOverlayPress}
-                />
+                {/* This condition allows for touches to work on the initial 3 frames
+                where the trick of getting the elements heights is done */}

```

Comment:
What I don't fully get is this part, let's talk about this.

Reply:
At the middle of writing an explanation, I remembered that this is not the case anymore. It was when I started implementing this component.

I fixed it again. The boolean condition still needs to be present, so that the overlay isn't in the component when the modal isn't visible. Either way, `this.state.visible` is enough